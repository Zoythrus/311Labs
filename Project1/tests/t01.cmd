  text1_1.txt text1_2.txt 
