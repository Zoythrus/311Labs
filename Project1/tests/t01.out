life is what happens when you are busy making other plans 
